### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by karenkrieger**
Documentation review

#### What problem is this solving?

General review of the `search-result` documentation (README.md file).

#### How to test it?

[Workspace](Link goes here!)

### CODE REVIEW

**Comment by karenkrieger** [comment]
Should I update the CHANGELOG file as well? I didn't add anything new, I just edited what was already in README.

File: docs/README.md [hunk 1]
```diff
@@ -26,47 +28,53 @@ In your theme's `manifest.json`, add the Search Result app as a dependency:
...
 | `total-products.v2`        | Displays the total amount of products found for that search. | 
 | `search-title.v2`         | Displays a title for the search that was done. |                                                                                              |
-| `search-fetch-more`         | Displays the "Show More" button. This button isn't rendered when the user is on the last page. |                                                                                              |
-| `search-fetch-previous`         | Displays the "Show Previous" button. This button isn't rendered when the user is on the first page. |                                                                                              |
-| `search-products-count-per-page`         | Displays the quantity of products currently on the page. |                                                                                              |
+| `search-fetch-more`         | Displays the **Show More** button. This button isn not rendered when the user is on the last page. |                                                                                              |
```
Comment (hiagolcm) [inline docs/README.md:54]:
```suggestion
| `search-fetch-more`         | Displays the **Show More** button. This button is not rendered when the user is on the last page. |                                                                                              |
```

**Review by hiagolcm — APPROVED** [review]
The documentation is much better. Nice job 🚀

File: docs/README.md (hunk 1 above)
Comment (karenkrieger) [reply docs/README.md:54]: Thank you!! ✨

**Review by PedroAntunesCosta — APPROVED** [review]
LGTM

**Review by PedroAntunesCosta — APPROVED** [review]
LGTM

**Pull request merged**